travvy-server/app.js

```diff
@@ -29,3 +29,13 @@ sequelize.sync().then(() => {
 })
 
 
+app.post('/signup',(req,res) =>{
+  res.send({
+    message: 'Successfully registered'
+  })
```

Comment:
these messages are perfect! now that theyre added, we'll know if the post request was made successfully or not